example/config/config.yml

```diff
@@ -28,3 +28,6 @@ scripts:
     output_file: density_temperature.png
     section: Density-Temperature
     title: Density-Temperature
+    additional_args:
+      cmap: cividis
```

Comment:
I'd really prefer if it we can use something here that doesn't include colour maps as an example - I don't want to encourage this.

Reply:
yes, no interest in changing `cmap`, just needed something to demonstrate functionality in the provided example plot. Could also have option to plot subgrid or hydro properties